### DISCUSSION TITLE
New posts alert removal?

### DISCUSSION THREAD

**Discussion by ChrisHisss** [typefully/minimal-twitter#161] [Q&A]
Hi there! 

I am not sure what this is called but after perusing the settings and searching the discussions, 
! (https://user-images.githubusercontent.com/124454635/216779379-7b7a5c5e-7d7a-4750-90f9-ff304edcc7e8.PNG)
I can't seem to find how to make it stop, it is so so so annoying. 

https://imgur.com/a/5XDRoFG

**Comment by ghost** [comment]
Ah yes we've discussed with, we have some potential solutions, possibly hiding this function and automatically loading new posts. 

cc @linuz90
